implementations/python/sugar-lift-py-tests/tests/test_callable_constant_coordinate.py

```
@@ -0,0 +1,37 @@
+
+
+def test_import_alias_is_already_a_named_constant_coordinate() -> None:
+    term = ImportAliasValue("datetime", "dt").to_term(owner="test")
+    assert term == ctor(
+        "python:import_alias", [str_const("dt"), str_const("datetime")]
+    )
+
+
+def test_actually_undeclared_variable_still_refuses_scope() -> None:
+    formula = formula_from_ir(eq(make_var("ghost"), num(1)), var_sorts={"ghost": IntSort()})
+    with pytest.raises(FactoryPanic, match="illegal free var.*ghost"):
```

Ruff (0.15.20)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)